GVRf/Framework/src/org/gearvrf/GVRActivity.java

```diff
@@ -45,6 +45,7 @@
 
     private GVRViewManager mGVRViewManager = null;
     private GVRCamera mCamera;
+    private boolean forceMonoscopic = false;
```

Comment:
Much as I dislike the `'m'` prefix, I think we should be internally consistent: when you add a new field to a block of fields, you probably should honor the local style.